dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/status/GetOrderStatusSpec.scala

```diff
@@ -31,36 +31,36 @@ class GetOrderStatusSpec extends MatcherSuiteBase with TableDrivenPropertyChecks
       val o = mkOrder(alice, wavesUsdPair, BUY, 10.waves, 2.usd)
 
       withClue(" - not found") {
-        validate200Json(dex1.rawApi.getOrderStatus(o)).status should be(Status.NotFound)
+        validate200Json(dex1.rawApi.orderStatusByAssetPairAndId(o)).status should be(Status.NotFound)
```

Comment:
This is my mistake. This method should be `getOrderStatusByAssetPairAndId` :( Sorry